content/using-sagemaker-managed-spot-training/before/aws_event/_index.md

```diff
@@ -0,0 +1,25 @@
++++
+title = "...At an AWS event"
+chapter = true
+weight = 5
++++
+
+### Running the workshop at an AWS Event
+
+{{% notice warning %}}
+Only complete this section if you are at an AWS hosted event (such as re:Invent, public workshop, Immersion Day, or any other event hosted by an AWS employee). If you are running the workshop on your own, go to: [Start the workshop on your own](/running-amazon-ec2-workloads-at-scale/before/self_paced.html).
```

Comment:
broken link

Reply:
This link seems to be broken locally on other workshops as well. Seems to have to do with the {{% notice warning %}} not rendering properly on local or off of a branch.